### PR TITLE
Fix Xcode 12 compatibility

### PR DESCRIPTION
**Pull request by jinshin1013**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fixes the Xcode 12 compatibility issue. More info can be found here https://github.com/facebook/react-native/issues/29633#issuecomment-694187116

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

**Comment by jinshin1013** [comment]
I'm not sure why the build is failing 🧐

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+
+#### iOS
+
+* **[Fix]** Fix Xcode 12 compatibility.
+
+### App Center Push
+
+#### iOS
+
+* **[Fix]** Fix Xcode 12 compatibility.
```
Comment (MatkovIvan) [inline CHANGELOG.md:27]:
```suggestion
___
```

**Comment by MatkovIvan** [comment]
Thanks for thr contribution! The changes looks good for me, but waiting for another look from teammates

**Comment by russelarms** [comment]
Thanks for the contribution, @jinshin1013. I've tested it on iOS, works fine.

**Review by russelarms — APPROVED** [review]

**Pull request merged**